src/main/java/org/springframework/data/elasticsearch/client/NodeClientFactoryBean.java

```diff
@@ -99,10 +98,10 @@ public boolean isSingleton() {
 	@Override
 	public void afterPropertiesSet() throws Exception {
 
-		nodeClient = (NodeClient) new TestNode(Settings.builder().put(loadConfig()).put("transport.type", "netty4")
-				.put("http.type", "netty4").put("path.home", this.pathHome).put("path.data", this.pathData)
-				.put("cluster.name", this.clusterName).put("node.max_local_storage_nodes", 100).build(),
-				asList(Netty4Plugin.class)).start().client();
+		node = new TestNode(Settings.builder().put(loadConfig()).put("transport.type", "netty4").put("http.type", "netty4")
```

Comment:
Nit: Wouldn't it be good to move the settings to new lines?

```suggestion
		Settings settings = Settings.builder() //
                   .put(loadConfig()) //
                   .put("transport.type", "netty4") //
                   .put("http.type", "netty4") //
                   .put("path.home", this.pathHome) //
                   .put("path.data", this.pathData) //
                   .put("cluster.name", this.clusterName) //
                   .put("node.max_local_storage_nodes", 100) //
                   .build();

		node = new TestNode(settings, Collections.singletonList(Netty4Plugin.class))
```